src/rp2_common/pico_bootrom/include/pico/bootrom.h

```diff
@@ -1101,6 +1101,9 @@ int rom_add_flash_runtime_partition(uint32_t start_offset, uint32_t size, uint32
  * a main image TBYB boot. It requires the same minimum workarea size as `rom_pick_ab_partition`.
  * \see rom_pick_ab_partition()
  * 
+ * This should be used instead of `rom_pick_ab_partition` when performing a Flash Update Boot before calling `explicit_buy`, and can still be used without
```

Comment:
Commenting on a wider scope:

* any in progress update or TBYB boot" perhaps "any in progress Flash Update boot or TBYB boot" (note boot repeated), though a TBYB is a flash update boot, so maybe TBYB operation IDK
* I'd put this function right after rom_pick_ab_partition in the header
* we should use \ref to refer to other functions not `func` (cc @lurch)
* arguably "normally" is "always" so really you could lose the word
* bit of a weird '-' in the middle of the For example sentence
* erase 2 partitions -> erase two partitions
* It also checks" It is a big dangly here (perhaps this function)